.gitignore

```diff
@@ -34,8 +34,17 @@ build/Release
 node_modules
 jspm_packages
 
+# Npm lock files
+package-lock.json
```

Comment:
this gets checked in and it shouldn't be ignored.  If this changes you should see the diff